import_models_to_verta/mlflow_databricks_bulk_import_example.ipynb

```diff
@@ -0,0 +1,341 @@
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
+   "source": [
+    "# os.environ[\"MLFLOW_TRACKING_URI\"] = \"\"\n",
+    "# os.environ[\"DATABRICKS_HOST\"] = \"\"\n",
+    "# os.environ[\"DATABRICKS_TOKEN\"] = \"\"\n",
+    "\n",
+    "mlflow_client = MlflowClient()"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## Fetch MLflow Models\n",
+    "Get a list of models registered in MLFLow and print them out for inspection.  This entire result set can be passed along to be imported to Verta, or pared down to any desired subset.  This can be achieved by passing search criteria to the `search_registered_models` method (see [MLflow API Docs](https://mlflow.org/docs/latest/python_api/mlflow.client.html?highlight=search_registered_models#mlflow.client.MlflowClient.search_registered_models)), or by filtering the results after they are returned."
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
+   "source": [
+    "mlflow_models: List[MLflow_RegisteredModel]= list()\n",
+    "\n",
+    "result: PagedList = mlflow_client.search_registered_models()\n",
+    "mlflow_models += result.to_list()\n",
+    "while result.token:  # handle paginated results\n",
+    "    result: PagedList = mlflow_client.search_registered_models(\n",
+    "    page_token=result.token\n",
+    "    )\n",
+    "    mlflow_models += result.to_list()\n",
+    "\n",
+    "mlflow_models.sort(key=lambda x: x.name)\n",
+    "for mv in mlflow_models:\n",
+    "    print(f\"name={mv.name}; version={mv.version}\")"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "Reduce the list if desired"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
+   "source": [
+    "# Example: filter out models with \"test\" in the name\n",
+    "models_for_import = [model for model in mlflow_models if \"test\" not in model.name]"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "# Import Models to Verta"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
+   "source": [
+    "# These are the names of files that Verta uses internally and cannot be overwritten\n",
+    "protected_file_names = [\n",
+    "    \"custom_modules\",\n",
+    "    \"model\",\n",
+    "    \"model.pkl\",\n",
+    "    \"model_api.json\",\n",
+    "    \"requirements.txt\",\n",
+    "    \"train_data\",\n",
+    "    \"tf_saved_model\",\n",
+    "    \"setup_script\",\n",
+    "]"
```

Comment:
I'm personally not a huge fan of putting our internally-reserved keys in user-consumable code like this. We already have a check for `"model"` and the rest of these are obscure enough that I think the risk is low enough to omit this 😬

Reply:
Removed